Index: software-center/build/share/applications/unbranded-software-center.desktop
===================================================================
--- software-center.orig/build/share/applications/unbranded-software-center.desktop	2012-01-03 13:58:28.000000000 -0430
+++ software-center/build/share/applications/unbranded-software-center.desktop	2012-01-03 14:01:18.000000000 -0430
@@ -159,7 +159,8 @@
 Comment[en_CA]=Lets you choose from thousands of free applications available for your system 
 Comment[en_GB]=Lets you choose from thousands of free applications available for your system 
 Comment[fr]=Choisissez parmi les milliers d'applications libres disponibles pour votre système 
-Comment[pt_BR]=Permite-lhe escolher entre milhares de aplicativos livres disponíveis para o seu sistema 
+Comment[pt_BR]=Permite-lhe escolher entre milhares de aplicativos livres disponíveis para o seu sistema
+Comment[es_VE]=Permirte escojer entre miles de aplicaciones libres disponibles para su sistema. 
 Exec=/usr/bin/software-center
 Icon=softwarecenter
 Terminal=false
