Index: software-center-5.1.2debian3.1/softwarecenter/ui/gtk3/widgets/exhibits.py
===================================================================
--- software-center-5.1.2debian3.1.orig/softwarecenter/ui/gtk3/widgets/exhibits.py	2013-03-19 14:44:32.730479285 -0430
+++ software-center-5.1.2debian3.1/softwarecenter/ui/gtk3/widgets/exhibits.py	2013-03-19 14:49:34.062468847 -0430
@@ -79,14 +79,14 @@
 
     def __init__(self):
         self.id = 0
-        self.package_names = "armagetronad,calibre,cheese,homebank,stellarium,gimp,inkscape,blender,audacity,gufw,frozen-bubble,fretsonfire,moovida,liferea,arista,gtg,freeciv-client-gtk,openshot,supertuxkart,tumiki-fighters,tuxpaint,webservice-office-zoho"
+        self.package_names = "0ad,vlc,darktable,kmag,ekiga,planner,canaima-desarrollador,canaima-semilla,inkscape,geany,emesene,deluge,sound-juicer,cheese,openproj,dia,gnome-bluetooth,armagetronad,calibre,cheese,homebank,stellarium,gimp,inkscape,blender,audacity,gufw,frozen-bubble,fretsonfire,moovida,liferea,arista,gtg,freeciv-client-gtk,openshot,supertuxkart,tumiki-fighters,tuxpaint,webservice-office-zoho"
         self.title_translated = _("Our star apps")
         self.published = True
         self.banner_url = "file:%s" % (os.path.abspath(os.path.join(softwarecenter.paths.datadir, "default_banner/fallback.png")))
         self.html = EXHIBIT_HTML % { 
             'banner_url' : self.banner_url,
-            'title' : _("Our star apps"),
-            'subtitle' : _("Come and explore our favourites"),
+            'title' : _(" "),
+            'subtitle' : _(" "),
       }
         # we should extract this automatically from the html
         #self.atk_name = _("Default Banner")
